evaluation/tests/interface_tests/run.sh

```diff
@@ -328,7 +339,7 @@ test_IFS()
 }
 
 ## We run all tests composed with && to exit on the first that fails
-if [ "$#" -eq 0 ]; then
+if [ "$#" -eq 0 ] || [ "$test_mode" = "bash" ]; then
```

Comment:
I would really remove all these control flow checks and make sure that all tests always run both with and without --bash.